docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-AGGREGATE.xml

```diff
@@ -15,14 +15,18 @@
       <primary>RIGHT1</primary>
     </indexterm>,RIGHT2<indexterm>
       <primary>RIGHT2</primary>
-    </indexterm>) ] [,</emphasis> <emphasis>groupingfields</emphasis>
-  <emphasis role="bold">] [, LOCAL<indexterm>
+    </indexterm>) |</emphasis> <emphasis>groupingfields</emphasis> <emphasis
```

Comment:
I think the original was possibly clearer, but I'm not significant enough difference to stop it being merged.